Installation.md

```diff
@@ -0,0 +1,267 @@
+10. Run the test suite with `rvmsudo rake spec`.
+11. Get pghero stats enabled, add to postgresql.conf following lines `shared_preload_libraries = 'pg_stat_statements'` and `pg_stat_statements.track = all`
+
+After you get the test suite to pas you can set up [background services](#background-services) and [startup](#startup) the server.
+
+#### Installation ####
+##### Set Java, Ruby and PhantomJS versions to be installed #####
+
+```
+$ JAVA_VERSION=7
+$ RUBY_VERSION=2.2.0
+$ PHANTOM_JS="phantomjs-1.9.8-linux-x86_64"
+```
+
+##### Install dependencies #####
+
+Update your package list with
+```
+$ sudo apt-get update
+```
+
+##### Java installation #####
+
+To install OpenJDK 7 run
+```
+$ sudo apt-get install openjdk-7-jdk
+```
+If you want to install the official Oracle JDK, you need to add an ppa repository. You can install Oracle JDK with the following commands
+```
+$ sudo add-apt-repository ppa:webupd8team/java
+$ sudo apt-get update
+$ sudo apt-get install oracle-java$JAVA_VERSION-installer
+```
+
+##### Install dependencies #####
+
+TMC-server dependencies
+```
+$ sudo apt-get install git build-essential zip unzip imagemagick maven make bc xvfb postgresql postgresql-contrib
+$ sudo apt-get install chrpath libssl-dev libxft-dev \
+  libfreetype6 libfreetype6-dev libfontconfig1 libfontconfig1-dev
+```
+Ruby dependencies
+```
+$ sudo apt-get install git-core curl zlib1g-dev libssl-dev libreadline-dev libyaml-dev libsqlite3-dev sqlite3 libxml2-dev libxslt1-dev libcurl4-openssl-dev python-software-properties
+```
+RVM dependencies
+```
+$ sudo apt-get install libgdbm-dev libncurses5-dev automake libtool bison libffi-dev
+```
+TMC-sandbox dependencies
+```
+$ sudo apt-get install squashfs-tools multistrap e2fsprogs e2tools
+```
+
+##### Installing PhantomJS #####
+
+Download PhantomJS from their [website](http://phantomjs.org/)
+```
+$ wget https://bitbucket.org/ariya/phantomjs/downloads/$PHANTOM_JS.tar.bz2
+```
+After download finishes extract the files and create symlinks
+```
+$ tar -xvjf $PHANTOM_JS.tar.bz2
+$ sudo mv $PHANTOM_JS /usr/local/share
+$ sudo ln -s /usr/local/share/$PHANTOM_JS/bin/phantomjs /usr/local/bin
+$ rm $PHANTOM_JS.tar.bz2
+```
+Check that PhantomJS is installed correctly
+```
+$ phantomjs --version
+```
+
+### Install ruby via RVM ###
+#### Install RVM as multi-user install ####
+
+```
+$ gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
+$ \curl -L https://get.rvm.io | sudo bash -s stable
+```
+
+After installation finishes add new user and setup groups
+
+```
+$ sudo adduser tmc
+$ sudo addgroup rvm
+$ sudo usermod -a -G rvm tmc
+$ sudo /usr/local/rvm/bin/rvm install $RUBY_VERSION
+$ sudo /usr/local/rvm/bin/rvm use $RUBY_VERSION --default
+```
+
+Setup ruby path
+
+```
+$ echo 'export PATH=$PATH:/usr/local/rvm/rubies/ruby-$RUBY_VERSION/bin' | sudo tee -a etc/profile.d/ruby.sh
+```
```

Comment:
Tell the user to source that file or restart the terminal (or is a relogin needed on desktops?)